### ISSUE TITLE
Valves with transform 1,-1 cannot be controlled from details dialogue

### ISSUE THREAD

**Issue by adambrown1** [AG-Schumann/Doberman#183]
For some valves it is useful to use a transform 1,-1 to invert the status, so that doberman shows the real state of the valve. For example the two next to the getter, which are NO valves. This means that when the mduino output is high, the solenoid valve opens allowing air to the pneumatic valve which then closes.

For these, the sensor details dialogue shows the "wrong" button: since the value of the sensor is 1 when the valve is open, it shows the close button and sends the commands set XXX 0. But to close the valve, we need to set the mduino output high. You have to send the command manually and have to know to send the opposite command of what you want.

In short, when the transform is used as 1,-1, the button should send the opposite output to normal.

**Comment by adambrown1** [comment]
Do you have a concrete suggestion?

**Comment by darrylmasson** [comment]
We add a "NC" field to the sensor doc for relevant switches, and then invert the command.
